packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -308,6 +311,8 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
       highlightedIndex: this.props.highlightFirstItemOnOpen ? 0 : null,
       searchQuery: search ? '' : undefined,
       value: multiple ? [] : null,
+      isFromKeyboardItem: false,
```

Comment:
nit: may suggest to rename it to something like `itemIsFromKeyboard`, to better reflect semantics of this field - mainly due to `keyboardItem` is quite confusing